src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.minutes {
+  position: absolute;
+  bottom: 50%;
+
+  height: 20vmin;
+  width: 3vmin;
+  background-color: #0700ff;
+
+  animation: rotate-bottom 3600s steps(60, end) infinite forwards;
+
+  &--speed-up {
+    animation: rotate-bottom 600s steps(60, end) infinite forwards;
+  }
+}
+
+.seconds {
+  position: absolute;
+  bottom: 50%;
+
+  height: 38vmin;
+  width: 1.5vmin;
+  background-color: #2c8000;
+
+  animation: rotate-bottom 60s linear infinite;
+
+  &--speed-up {
+    animation: rotate-bottom 10s linear infinite;
+  }
+}
+
+.center {
+  position: absolute;
+
+  width: 5vmin;
+  height: 5vmin;
+  background-color: #f6a603;
+  border-radius: 50%;
+}
+
+@keyframes rotate-bottom {
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), each BEM block should be in a separate file. The keyframes `rotate-bottom` should be defined in a separate SCSS partial and then imported into this file.